OrbitCore/SymbolHelper.cpp

```diff
@@ -168,27 +168,21 @@ bool SymbolHelper::LoadSymbolsUsingSymbolsFile(
   return FindAndLoadSymbols(module, symbols_file_directories_);
 }
 
-void SymbolHelper::LoadSymbolsFromDebugInfo(
-    std::shared_ptr<Module> module, const ModuleDebugInfo& module_info) const {
-  module->m_Pdb =
-      std::make_shared<Pdb>(module->m_AddressStart, module_info.load_bias,
-                            module_info.m_PdbName, module->m_FullName);
-
-  for (const auto& function : module_info.m_Functions) {
-    module->m_Pdb->AddFunction(function);
+void SymbolHelper::LoadSymbolsIntoModule(
+    std::shared_ptr<Module> module,
```

Comment:
```suggestion
    const std::shared_ptr<Module>& module,
```

To save myself some clang-tidy warnings.

Reply:
done